### PR TITLE
support 'add' and 'remove' options for dolt_backup()

### PR DESCRIPTION
**Pull request by jennifersp**

### CODE REVIEW

File: integration-tests/bats/sql-backup.bats [hunk 1]
```diff
@@ -17,21 +17,63 @@ teardown() {
...
-    [ "$status" -ne 0 ]
+    mkdir the_backup
+    run dolt sql -q "call dolt_backup('add', 'hostedapidb-0', 'file://./the_backup')"
+    [ "$status" -eq 0 ]
+    run dolt backup -v
+    [ "$status" -eq 0 ]
+    [[ "$output" =~ "the_backup" ]] || false
+}
+
+@test "sql-backup: dolt_backup ass cannot add remote with address of existing backup" {
```
Comment (fulghum) [inline integration-tests/bats/sql-backup.bats:28]: is `dolt_backup ass` a new command? 🤔 

j/k 😆

File: integration-tests/bats/sql-backup.bats [hunk 2]
```diff
@@ -17,21 +17,63 @@ teardown() {
...
+    [ "$status" -eq 0 ]
+    [ "${#lines[@]}" -eq 1 ]
+    [[ "$output" =~ "bac1" ]] || false
+
+    dolt backup remove bac1
+
+    run dolt backup -v
+    [ "$status" -eq 0 ]
+    [ "${#lines[@]}" -eq 0 ]
+
```
Comment (fulghum) [inline integration-tests/bats/sql-backup.bats:49]: (nit)

```suggestion
```

File: integration-tests/bats/sql-backup.bats [hunk 3]
```diff
@@ -17,21 +17,63 @@ teardown() {
...
+    [ "${#lines[@]}" -eq 1 ]
+    [[ "$output" =~ "bac1" ]] || false
+
+    dolt backup rm bac1
+
+    run dolt backup -v
+    [ "$status" -eq 0 ]
+    [ "${#lines[@]}" -eq 0 ]
+    [[ ! "$output" =~ "bac1" ]] || false
 }
```
Comment (fulghum) [inline integration-tests/bats/sql-backup.bats:78]: (minor) Might be nice to add in a quick test for the error case when an http/https URL is provided.

**Review by fulghum — APPROVED** [review]
Thanks for knocking this one out! Looks great; just a couple really minor comments from me.

File: integration-tests/bats/sql-backup.bats (hunk 1 above)
Comment (jennifersp) [reply integration-tests/bats/sql-backup.bats:28]: Oops 😂

**Pull request merged**